glass/galaxies.py

```diff
@@ -117,43 +117,44 @@ def redshifts_from_nz(
         rng = np.random.default_rng()
 
     # bring inputs' leading axes into common shape
-    dims, count, z, nz = broadcast_leading_axes((count, 0), (z, 1), (nz, 1))  # type: ignore[no-untyped-call]
+    dims, *rest = broadcast_leading_axes((count, 0), (z, 1), (nz, 1))
+    count_out, z_out, nz_out = rest
```

Comment:
This change should not be necessary if `broadcast_leading_axes()` is typed correctly

Reply:
Could we leave this now for now and fix properly in #386?